doc/man7/OSSL_PROVIDER-FIPS.pod

```diff
@@ -140,6 +140,8 @@ This is an unapproved algorithm.
 
 =item X448, see L<EVP_KEYEXCH-X448(7)>
 
+=item ML-KEM, see L<EVP_KEM-ML-KEM(7)>
+
```

Comment:
Should this be expanded into all the bit variants?

Reply:
I think the single reference is sufficient, but I'm open to listing them all.
The motivation to keep it brief is to not clutter the user's screen with duplicate information.

Reply:
@t8m Would you prefer all the variants here sufficiently to request it be changed?